main.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ * Result.cpp
+ *
+ *  Created on: 2021/10/02
+ *      Author: hommyo
+ */
+
+#include "./lib/Result.hpp"
+
+#include <stdio.h>
+#include <string>
+#include <chrono>
+#include <assert.h>
+
+using namespace std;
+
+void test_Statistics_Welfords_online_algorithm();
+
+int main(void)
+{
+	printf("example run\n");
+
+	test_Statistics_Welfords_online_algorithm();
+
+	return 0;
+}
+
+/* # 基本統計量構造体
+ *   基本統計量計算のテスト
+ *
+ */
+void test_Statistics_Welfords_online_algorithm()
+{
+	//Statistics関数使用時のテスト
+	vector<Result> results1
```

Comment:
```suggestion
	vector<Result> results
```
これどう？わざわざ数字添えなくてもいいかなって思った